### DISCUSSION TITLE
[Symfony 5.3] form_div_layout.html.twig : How to implement custom widget form now ?

### DISCUSSION THREAD

**Discussion by meiyasan** [symfony/symfony#42531] [Q&A]
Hello guys, 

I have been used to customise my form layout by introducing new blocks in the form_div_layout.html.twig file.
Now since I updated from Symfony 5.2 to 5.3, my custom widgets are not recognised.
Any idea of to traceback the issue ?

The documentation page seems to be much smaller now:
https://symfony.com/doc/current/form/form_customization.html
https://symfony.com/doc/4.0/form/form_customization.html#how-to-customize-an-individual-field (Before)

For instance I had designed a SlugType, the corresponding widget was as below. Now the field appears to be like a normal text bar. Any idea ?

```twig
{% block slug_widget %}

        {% set attr = attr|merge({
        'data-slug-field': '',
        'data-target': form.parent[target].vars.id
    }) %}

        <div class="input-group">
                {{ block('form_widget') }}
                <div class="input-group-append">
                        <button class="btn btn-outline-secondary" type="button">
                                <i class="fas fa-lock fa-fw"></i>
                        </button>
                </div>
        </div>
{% endblock %}
```

Cheers, Marco

**Comment by lyrixx** [comment]
* Did you call `{% form_theme form _self %}`
* Did you update the name of your `SlugType` ? You can find the block name you must use in the profiler > form > view vars > block_prefixes (something like that)

**Comment by romaricdrigon** [comment]
I don't think there was any major change in that part, on my side I didn't have any change regarding form themes while upgrading. Maybe you changed something else? Maybe `config/packages/twig.yaml` was modified?  
Also check form "block prefixes", as said earlier by lyrixx.
